### ISSUE TITLE
[Bug]: Can not decrypt messages

### ISSUE THREAD

**Issue by CubicrootXYZ** [CubicrootXYZ/RemindMe#278]
### Describe the bug

Receiving messages from encrypted rooms fails with: 
```
can not handle event: failed to decrypt megolm event: no session with given ID found
```

### How sever is this bug?

HIGH (main user interaction affected)

### Version

v2.0.0-beta4

### Checklist

- [X] I was able to reproduce the bug with the newest version of the bot
- [X] I confirmed that there is no other issue reporting the same bug

**Comment by CubicrootXYZ** [comment]
Forgot to mount the `olm.db`, with that one mounted there is no error visible, yet the logs do not show any incoming messages for that room.

**Comment by CubicrootXYZ** [comment]
The bot seems to not follow invites for rooms it got kicked out previously.

**Comment by CubicrootXYZ** [comment]
Not sure if it is worth keeping this feature updated and running. Encryption is very difficult and any small issue might break existing rooms. Currently my gut feeling tells me that this feature simply is not worth the effort I'd need to put into it.

**Comment by CubicrootXYZ** [comment]
Getting rid of the E2EE would also get rid of the `olm.db` sqlite file and all the CGO dependencies. That would dramatically lower build times and image sizes. 🤔

**Comment by CubicrootXYZ** [comment]
Solved by removing e2ee support in #298
